### PR TITLE
fix(ESSNTL-3831): Links without page refresh

### PR DESCRIPTION
**Pull request by AsToNlele**
[ESSNTL-3831](https://issues.redhat.com/browse/ESSNTL-3831)

Links now work without full page refresh, so eg. global filter should stay applied.

How to test:
- Make sure all links to our apps are dynamic

### CODE REVIEW

**Review by bastilian — APPROVED** [review]
NICE! Thank you @AsToNlele!

File: src/PresentationalComponents/ZeroState/AppZeroState.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ const AppZeroState = ({ app, customInstructions, customButton, customText, custo
                     customTitle={customTitle}
                     appId={appId}
                 />
+                xd
```
Comment (mkholjuraev) [inline src/PresentationalComponents/ZeroState/AppZeroState.js:22]: Before this gets merged, let's remove this.

**Review by mkholjuraev — CHANGES_REQUESTED** [review]
It seems there is a small leftover :). For visibility, I am requesting changes

File: src/PresentationalComponents/ZeroState/AppZeroState.js (hunk 1 above)
Comment (bastilian) [reply src/PresentationalComponents/ZeroState/AppZeroState.js:22]: Nice catch. :D

Comment (AsToNlele) [reply src/PresentationalComponents/ZeroState/AppZeroState.js]: Thanks this one debugging tool got away! :smile:

**Comment by AsToNlele** [comment]
Removed the `xd`, thanks for reviews!

**Pull request merged**